packages/core/src/accordion/accordion.stories.mdx

```diff
@@ -1,7 +1,7 @@
 import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
 import { html, LitElement } from 'lit-element';
 
-<Meta title="Components (Preview)/Accordion" />
+<Meta title="Components/Accordion" />
```

Comment:
If we are moving accordion out of Preview, should we remove the beta image from the nav on the website?

Reply:
Yeah, I think that will be part of a larger website nav update pr